### PR TITLE
Upgrade to Decidim v0.21

### PR DESCRIPTION
**Pull request by LauraJaimeC**
Changelog: https://github.com/decidim/decidim/blob/release/0.21-stable/CHANGELOG.md

- Update Decidim to v0.21
- Update Puma
- Remove unnecesary time zone configuration

!!!!!
This PR depends of https://github.com/CodiTramuntana/decidim-erc-crm_authenticable/pull/15
It is necessary to change the Gemfile when this PR will merge.

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -17,7 +17,6 @@ class Application < Rails::Application
     config.load_defaults 5.1
 
     # Default app time zone
-    config.time_zone = "Madrid"
     config.active_record.default_timezone = :local
     config.active_record.time_zone_aware_attributes = false
```
Comment (tramuntanal) [inline config/application.rb:21]: Maybe we should remove all these lines?

File: config/initializers/decidim.rb [hunk 2]
```diff
@@ -15,11 +15,10 @@
...
   config.track_newsletter_links = true
-  config.geocoder = {
-    static_map_url: 'https://image.maps.cit.api.here.com/mia/1.6/mapview',
-    here_app_id: Rails.application.secrets.geocoder[:here_app_id],
-    here_app_code: Rails.application.secrets.geocoder[:here_app_code]
-  }
+  # config.geocoder = {
+  #   static_map_url: "https://image.maps.ls.hereapi.com/mia/1.6/mapview",
+  #   here_api_key: Rails.application.secrets.geocoder[:here_api_key]
+  # }
```
Comment (tramuntanal) [inline config/initializers/decidim.rb:21]: it seems that geocoding was enabled, why are we commenting this configuration?

File: config/secrets.yml [hunk 3]
```diff
@@ -26,8 +26,7 @@ default: &default
       client_id: <%= ENV["OMNIAUTH_GOOGLE_CLIENT_ID"] %>
       client_secret: <%= ENV["OMNIAUTH_GOOGLE_CLIENT_SECRET"] %>
   geocoder:
-    here_app_id: <%= ENV["GEOCODER_LOOKUP_APP_ID"] %>
-    here_app_code: <%= ENV["GEOCODER_LOOKUP_APP_CODE"] %>
+    here_api_key: <%= ENV["GEOCODER_LOOKUP_API_KEY"] %>
```
Comment (tramuntanal) [inline config/secrets.yml:29]: do we have the new api_key? we will need it when deploying..

File: config/locales/ca.yml [hunk 4]
```diff
@@ -3,6 +3,10 @@ ca:
   decidim:
     actions:
       export_amendments: Exportar esmenes
+    admin:
+      filters:
+        state_eq:
+          label: Estat
```
Comment (tramuntanal) [inline config/locales/ca.yml:9]: we must check why we need this translations

File: docs/upgrades.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+## From v0.19 to v.0.20
```
Comment (tramuntanal) [inline docs/upgrades.md:1]: why are we adding this docs now if it were related to v0.20 which is already deployed?

**Review by tramuntanal — COMMENTED** [review]
Thanks  @LauraJaimeC 
I've found some changes that it will be best to comment

File: docs/upgrades.md (hunk 5 above)
Comment (LauraJaimeC) [reply docs/upgrades.md]: I had to do a master rebase to have the changes with the upgrade to v0.20

File: config/initializers/decidim.rb (hunk 2 above)
Comment (LauraJaimeC) [reply config/initializers/decidim.rb]: :thinking:

Comment (tramuntanal) [reply config/initializers/decidim.rb]: The keys were not configured so we can keep it commented :+1:

File: config/secrets.yml (hunk 3 above)
Comment (LauraJaimeC) [reply config/secrets.yml]: I have checked again and in PROD there are no keys for this :)

**Comment by LauraJaimeC** [comment]
@tramuntanal  you have to check this PR before merge because I have to change the Gemfile
https://github.com/CodiTramuntana/decidim-erc-crm_authenticable/pull/15

**Comment by tramuntanal** [comment]
Ok @LauraJaimeC , done! and approved.

**Review by tramuntanal — APPROVED** [review]
Good!

**Pull request merged**